app/controllers/searches_controller.rb

```diff
@@ -168,7 +168,7 @@ def common_friends
   def common_followers
     @user_items = build_user_items(@searched_tw_user.common_followers(current_user.twitter_user))
     @graph = @searched_tw_user.common_followers_graph(current_user.twitter_user)
-    @tweet_text = view_context.common_followers_text(@user_items.slice(0, 3).map{|i| i[:target] }, @searched_tw_user, @user_items.size - 3)
+    @tweet_text = view_context.common_followers_text(@user_items.slice(0, 3).map { |i| i[:target] }, @searched_tw_user, @user_items.size - 3)
```

Comment:
__[rubocop]__
* Line is too long. [128/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)